src/Microsoft.ML.Data/Model/Onnx/OnnxContext.cs

```diff
@@ -98,5 +98,56 @@ public abstract class OnnxContext
         /// <returns>A node added to the in-progress ONNX graph, that attributes can be set on</returns>
         public OnnxNode CreateNode(string opType, string input, string output, string name, string domain = null)
             => CreateNode(opType, new[] { input }, new[] { output }, name, domain);
+
+        /// <summary>
+        /// Call this function can declare a global float
+        /// </summary>
+        /// <param name="value">The float number which is going to be added</param>
+        /// <param name="name">A string used as a seed to generate this initializer's name in the ONNX graph.</param>
+        /// <returns>The initializer's ONNX name</returns>
+        public abstract string AddInitializer(float value, string name = null);
+
+        /// <summary>
+        /// Call this function can declare a global int64
```

Comment:
>int64 [](start = 52, length = 5)

long

Reply:
The long is C# name. We need to use ONNX naming in ONNX context.